Sources/Clappr_iOS/Classes/Base/Options.swift

```diff
@@ -11,6 +11,7 @@ public let kPlaybackNotSupportedMessage = "playbackNotSupportedMessage"
 public let kMimeType = "mimeType"
 public let kDefaultSubtitle = "defaultSubtitle"
 public let kDefaultAudioSource = "defaultAudioSource"
+public let kMinDvrOption = "minDvrTime"
```

Comment:
Hey @geisonfgf , I suggest change the name to `kMinDvrTime` to make it self explaining.

Reply:
web is using this name `minDvrSize`

Reply:
Hey guys, let's use the Player Web name.